tests/functional/test_matrix_service.py

```diff
@@ -43,32 +42,63 @@ class TestMatrixService(unittest.TestCase):
     def setUp(self):
         self.deployment_stage = os.environ['DEPLOYMENT_STAGE']
         self.api_url = f"https://{os.environ['API_HOST']}/v0"
+        self.res_dir = os.path.join(os.path.dirname(os.path.realpath(__file__)), "res")
         self.headers = {'Content-type': 'application/json', 'Accept': 'application/json'}
         self.verbose = True
         self.s3_file_system = s3fs.S3FileSystem(anon=False)
 
     def test_zarr_output_matrix_service(self):
-        request_id = self._post_matrix_service_request("zarr")
+        request_id = self._post_matrix_service_request(
+            INPUT_BUNDLE_IDS[self.deployment_stage], "zarr")
         WaitFor(self._poll_get_matrix_service_request, request_id)\
             .to_return_value(MatrixRequestStatus.COMPLETE.value, timeout_seconds=180)
-        self._analyze_zarr_matrix_results(request_id)
+        self._analyze_zarr_matrix_results(request_id, INPUT_BUNDLE_IDS[self.deployment_stage])
 
     def test_loom_output_matrix_service(self):
-        request_id = self._post_matrix_service_request("loom")
+        request_id = self._post_matrix_service_request(
+            INPUT_BUNDLE_IDS[self.deployment_stage], "loom")
         # timeout seconds is increased to 600 as batch may tak time to spin up spot instances for conversion.
+        WaitFor(self._poll_get_matrix_service_request, request_id)\
+            .to_return_value(MatrixRequestStatus.COMPLETE.value, timeout_seconds=180)
+        self._analyze_loom_matrix_results(request_id, INPUT_BUNDLE_IDS[self.deployment_stage])
+
+    @unittest.skip
+    def test_matrix_service_ss2_small(self):
```

Comment:
Actually this test I wrote is identical to Parth's `test_loom_output_matrix_service` (and broken). Can remove

Reply:
Okay, will remove